xml/docker_building_images.xml

```diff
@@ -8,68 +8,74 @@
  xmlns="http://docbook.org/ns/docbook" version="5.0"
  xmlns:xi="http://www.w3.org/2001/XInclude"
  xmlns:xlink="http://www.w3.org/1999/xlink">
- <title>Creating Custom Images</title>
+ <title>Creating Custom &docker; Images</title>
  <para>
-  For creating your custom image you need a base &docker; image of &slsa;. You
-  can use any of the pre-built &slsa; images that you can obtain as described
-  in <xref linkend="Customizing_Pre-build_Images"/>.
+  For creating your custom image, you need a base &docker; image of &slsa;.
+  You can use any of the pre-built &slsa; images that you can obtain as
```

Comment:
Proposal: You can use any of the pre-built images of &productname;. Download them as decribed in ```<xref .../>```.